File: 백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.cs
using System;

class Q17388
{
    static void Main(string[] args)
    {
        string[] s = Console.ReadLine().Split();
        int S = int.Parse(s[0]);
        int K = int.Parse(s[1]);
        int H = int.Parse(s[2]);
        int sum = S + K + H;
        int min = Math.Min(S, K);
		min = Math.Min(min, H);
        
        if (sum >= 100)
        {
            Console.WriteLine("OK");
        }
        else
        {
            if (min == S)
            {
                Console.WriteLine("Soongsil");
            }
            else if (min == K)
            {
                Console.WriteLine("Korea");
            }
            else
            {
                Console.WriteLine("Hanyang");
            }
        }
    }
}

File: 백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int S = sc.nextInt();
        int K = sc.nextInt();
        int H = sc.nextInt();
        int sum = S + K + H;
        int min = Math.min(S, K);
		min = Math.min(min, H);
        
        if (sum >= 100) {
            System.out.println("OK");
        } else {
            if (min == S) {
                System.out.println("Soongsil");
            } else if (min == K) {
                System.out.println("Korea");
            } else {
                System.out.println("Hanyang");
            }
        }
    }
}